docs/.vuepress/theme/styles/theme.scss

```diff
@@ -24,6 +25,7 @@ body {
 .cdr-doc-page-shell__side-navigation {
   flex: 0 0 $side-navigation-width;
   width: $side-navigation-width;
+  z-index: 120;
```

Comment:
Not disagreeing here, just wondering where 120 came from...

Reply:
100 is the z-index of the tabs, just need something higher than 100.

Reply:
👍